core/settings.py

```
@@ -208,6 +208,119 @@ def ld_loguru_sink(message):
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "django.middleware.cache.FetchFromCacheMiddleware",
+    "django_prometheus.middleware.PrometheusAfterMiddleware",
+    "common.middleware.RequestIDMiddleware",
+]
+
+
+def _insert_after_middleware(base, after_item, *new_items):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_insert_after_middleware`

(ANN202)